pages/js/frameworks.md

```diff
@@ -5,6 +5,14 @@ layout: docs
 sidenav: js
 ---
 # Frameworks
+
+**We recommend using [React]() for projects when possible.** As a team, we enjoy using React because it has a strong component model, enables easy server side rendering, and faciliates debugging by making the UI elements a function of the application state. React is most akin to the "view layer" in an MVC (Model-View-Controller) architecture and is frequently used with some other libraries. In our projects, we tend to use:
+
+* [Redux](https://www.npmjs.com/package/redux) for managing state
```

Comment:
Can I suggest adding "optional" as simpler webapps may not need to rely on Redux?